apple-pie/styles/style.css

```diff
@@ -17,8 +21,49 @@ body, h1, h2, h3, h4, h5, h6, p, ol, ul {
 
 /*  Set the font */
 body {
-  font-family: "Helvetica", sans-serif;
+  font-family: Verdana, Geneva, Tahoma, sans-serif
 }
-
 /* STYLES */
-/* Write your CSS below */
\ No newline at end of file
+/* Write your CSS below */
+#ps-heading {
+  text-align: center;
+  
+}
+.comments-paragraph {
+  text-align: center;
+}
+#ps-heading {
+  size: 100px;
+  text-align: center;
+}
+.comments-paragraph {
+  padding-top: 30px;
+}
+.image {
+  text-align: center;
+  height: 150px;
+}
+.h3 {
+  padding-bottom: 30px;
+}
+
+
+.container {
+  background-image: url("/apple-pie/images/apple-pie.jpg");
```

Comment:
This will work like this
```css
  background-image: url("./../images/apple-pie.jpg");
``` 
Since your are in the style.css file you need to get out the styles folder first and then into the images one